queue.h

```diff
@@ -6,9 +6,12 @@
 
 
 /*
- * How many items any queue_t instance can hold at a time.
+ * An enqueueable item with a generic data payload.
  */
-#define QUEUE_LENGTH 5
```

Comment:
Removing the `QUEUE_LENGTH` here is an unrelated change. Move into separate commit.

Reply:
Moved the removal of the `QUEUE_LENGTH` into the commit where the `queue_t` `struct` gets updated, as the array using the `QUEUE_LENGTH` for its declaration is also removed there.